pkg/apis/core/validation/validation.go

```diff
@@ -4050,15 +4048,6 @@ func validatePodIPs(pod *core.Pod) field.ErrorList {
 		if !dualStack || len(podIPs) > 2 {
 			allErrs = append(allErrs, field.Invalid(podIPsField, pod.Status.PodIPs, "may specify no more than one IP for each IP family"))
 		}
-
-		// There should be no duplicates in list of Pod.PodIPs
-		seen := sets.Set[string]{} // := make(map[string]int)
-		for i, podIP := range pod.Status.PodIPs {
-			if seen.Has(podIP.IP) {
-				allErrs = append(allErrs, field.Duplicate(podIPsField.Index(i), podIP))
-			}
-			seen.Insert(podIP.IP)
-		}
```

Comment:
you mean that this is covered by the dualstack check?

Reply:
there already is one: https://github.com/kubernetes/kubernetes/blob/v1.29.0/pkg/apis/core/validation/validation_test.go#L23173-L23179